client/layout/guided-tours/tours/main-tour.js

```diff
@@ -41,7 +41,7 @@ export const MainTour = makeTour(
 		<Step name="init" placement="right">
 			<p>
 				{
-					translate( "{{strong}}Need a hand?{{/strong}} We'd love to show you around the place," +
+					translate( "{{strong}}Need a hand?{{/strong}} We'd love to show you around the place, " +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '{{strong}}Need a hand?{{/strong}} We'd love to show you around the place,and give you some ideas for what to do next.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).